tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "strict": true,
+      "pretty": true,
+      "baseUrl": "./",
+      "outDir": "public",
+      "removeComments": true,
+      "experimentalDecorators": true,
+      "target": "es6",
+      "emitDecoratorMetadata": true,
+      "moduleResolution": "node",
+      "importHelpers": true,
+      "types": [
```

Comment:
It restricts included types definitions while all type definitions from @types subfolder should be used. I suggest we remove it. Default are better for us.